design-dimensions/Dimensions-with-General-Agreement.md

```diff
@@ -25,6 +25,11 @@ We’ve explored three main definitions of privacy:
 
 The community group has reached general agreement that the _Private Measurement Technical Specification MVP_ should use a definition of privacy based on differential privacy. This does not preclude the use of other privacy definitions in conjunction with differential privacy, however any proposal should aim to provide differential privacy guarantees.
 
+### Private measurement of single events
+The group has discussed the use-case of “differentially private measurement of single events”. Specifically, whether, given differential privacy protections, we would permit queries which ask about the outcome for a single event (did impression lead to a conversion, for instance). For more information about this, see https://github.com/patcg/docs-and-reports/issues/41 and the [agenda topic](https://github.com/patcg/meetings/issues/112) from the May 2023 meeting.
+
+We have reached consensus that these types of queries can be particularly sensitive, so we must take care when setting the proper [differential privacy scope](#privacy-unit--privacy-budget-scoping) and parameters (e.g., epsilon and delta) to ensure that user contributions are adequately protected. Any protection needs to cover people that submit many events. However, queries of this form will not necessarily be prohibited, although additional protections should be considered if differential privacy guarantees are deemed too weak.
```

Comment:
```suggestion
We have reached consensus that these types of queries can be particularly sensitive, so we must take care when setting the proper [differential privacy scope](#privacy-unit--privacy-budget-scoping) and parameters (e.g., epsilon and delta) to ensure that user contributions are adequately protected. A complication here is that the system might not know that two or more events are connected to a single user while the attacker might. Any protection needs to cover people that submit many events in such a scenario. 
Queries of this form will not necessarily be prohibited, although additional protections will be considered if differential privacy guarantees are deemed too weak.
```

Reply:
Suggested slightly stronger language to call out the threat explicitly, and make it clear that we _will_ work on additional protections if we think that DP is insufficient (and not just call it a day).

Reply:
I feel like this is concern is a topic for the [privacy unit](https://github.com/patcg/docs-and-reports/blob/main/design-dimensions/Dimensions-with-General-Agreement.md#privacy-unit--privacy-budget-scoping) section, which I'd like to have discussions on / consensus separately. Here I only want to say that the multiple events will be protected _within the privacy unit_ we choose. In discussion of the privacy unit, we can reference this issue. I would be happy to file an issue and we could discuss separately how we handle cases like this.

With that framing in mind this is my suggestion for how to clarify the language (and accept your should --> will wording change):
```suggestion
We have reached consensus that these types of queries can be particularly sensitive, so we must take care when setting the proper [differential privacy scope](#privacy-unit--privacy-budget-scoping) and parameters (e.g., epsilon and delta) to ensure that user contributions are adequately protected. Any protection needs to cover people that submit many events within the chosen privacy scope. However, queries of this form will not necessarily be prohibited, although additional protections will be considered if differential privacy guarantees are deemed too weak.
```

What do you think?

Reply:
I like @ShivanKaul's suggestion in its essence, but I have to agree with @csharrison on the execution.  Unfortunately, we don't have anything about people or users or whatever written down yet.  Presumably that is the next piece to cover.

Reply:
I guess that while single event queries can make the privacy risk worse (and we call this out), the idea that users might not be entirely captured by the privacy-unit is a different topic than what we're discussing here. Your wording seems fine.

Reply:
Thanks @martinthomson and @ShivanKaul . Applied the most recent edit.